File: CAPITULO 01/Java/ex3.java
import java.io.*;
import java.util.*;
import java.text.*;

public class ex3{
	public static void main (String[]args){
		float n1,p1,n2,p2,n3,p3,media;
		Scanner entrada;
		entrada = new Scanner(System.in);
		System.out.println("Digite a nota da n1 :");
		n1 = entrada.nextFloat();
		System.out.println("Informe o peo da n1 :");
		p1 = entrada.nextFloat();
		System.out.println("Digite a nota da n2 :");
		n2 = entrada.nextFloat();
		System.out.println("Informe o peso da n2 :");
		p2 = entrada.nextFloat(); 
		System.out.println("Digite a nota da n3 :");
		n3 = entrada.nextFloat();
		System.out.println("Informe o peso da n3 :");	
		p3 = entrada.nextFloat();
		media = (n1*p1)+(n2*p2)+(n3*p3);
		DecimalFormat cs;
		cs = new DecimalFormat("0.00");
		System.out.println(" A media pomderada das notas e de "+cs.format(media));
		
	}
}

File: CAPITULO 01/C++/ex3.cpp
#include <stdio.h>
int main()

{
	float n1,p1,n2,p2,n3,p3,media;
	printf("Digite a nota da n1 ");
	scanf("%d%*c",&n1);
	printf("Informe o peso da n1");
	scanf("%d%*c",&p1);
	printf("Digite a nota da n2 ");
	scanf("%d%*c",&n2);
	printf("Informe o peso da n2 ");
	scanf("%d%*c",&p2);
	printf("Digite a nota da n3 :");
	scanf("%d%*c",&n3);
	printf("Informe o peso da n3 :");
	scanf("%d%*c",&p3);
	media =  (n1*p1)+(n2*p2)+(n3*p3);
	printf ("A media ponderada das notas e de %2.3f");
	return 0;
}
